packages/next/test/fixtures/00-middleware-nested/vercel.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": 2,
+  "builds": [{ "src": "package.json", "use": "@vercel/next" }],
+  "probes": [
+    {
+      "path": "/redirect-me",
+      "status": 307,
+      "responseHeaders": {
+        "Location": "/from-next-config/"
+      },
+      "fetchOptions": {
+        "redirect": "manual"
+      }
+    }
+  ]
```

Comment:
I think I'm missing what is being tested here 😨 
I wonder if we should have a simpler middleware that adds a header or something and then make a probe that tries to use it? Seems like this only probe has nothing to do with the nested middleware